src/main/java/com/connexta/search/index/IndexManagerImpl.java

```diff
@@ -73,6 +92,49 @@ public void index(
     }
   }
 
+  @Override
+  public Set<String> query(String cql) throws IndexException {
```

Comment:
IndexException in a throws declaration is unnecessary since it extends RuntimeException